### DISCUSSION TITLE
There are some typing errors in doc.

### DISCUSSION THREAD

**Discussion by ghost** [v6d-io/v6d#234] [General]
Hi!

https://v6d.io/#integrate-with-kubernetes 
this is docs' url

There are two typing errors capability and routines maybe..
In the drivers section, 2,3 lines in second paragraph!

![image](https://user-images.githubusercontent.com/19821292/111567190-e556e280-87e1-11eb-8a2c-15db8dde0604.png)

thank you.

**Comment by sighingnow** [comment]
Thanks for letting us know that. Will fix that soon.

**Reply by sighingnow** [reply]
Fixed by #235.
